packages/cli-kit/src/public/node/cli.test.ts

```diff
@@ -75,6 +79,20 @@ describe('cli', () => {
     `)
   })
 
+  test('exits if running node version older than package.json engines field', async () => {
+    const launchCLI = vi.fn()
+    vi.mocked(findUpAndReadPackageJson).mockResolvedValue({content: {engines: {node: '>=22.0.0'}}} as any)
+    const run = runCLI({moduleURL: 'file:///test', development: false}, launchCLI, [], {}, {node: '21.9'} as any)
+    await expect(run).rejects.toThrow()
```

Comment:
Let's better ensure it's failing because of the right cause

```suggestion
    await expect(run).rejects.toThrow(/Node version/)
```